src/Console/Commands/EmitAllEvents.php

```diff
@@ -15,7 +15,8 @@ class EmitAllEvents extends Command
      */
     protected $signature = 'event-sourcing:emit-all
         {queueConnection=database : Queue connection to be used to send the messages}
-        {--unbufferedConnection= : Indicate the unbuffered connection (MySQL) for large amount of events}';
+        {--unbufferedConnection= : Indicate the unbuffered connection (MySQL) for large amount of events}
+        {--batchSize=1 : Indicate the amount of events to be sent per publish. Increase for higher throughput}';
```

Comment:
We can put a batch size higher, WDYT?

Reply:
There is no good default value, it depends on your use case, so I prefer to use the most standard value here.